demo/app/app.go

```diff
@@ -126,17 +126,17 @@ import (
 	upgradekeeper "github.com/cosmos/cosmos-sdk/x/upgrade/keeper"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 
-	"github.com/osmosis-labs/mesh-security-sdk/x/meshsecurity"
-	meshseckeeper "github.com/osmosis-labs/mesh-security-sdk/x/meshsecurity/keeper"
-	meshsectypes "github.com/osmosis-labs/mesh-security-sdk/x/meshsecurity/types"
+	babylon "github.com/babylonchain/babylon-sdk/x/babylon"
+	bbnkeeper "github.com/babylonchain/babylon-sdk/x/babylon/keeper"
+	bbntypes "github.com/babylonchain/babylon-sdk/x/babylon/types"
 )
 
-const appName = "MeshApp"
+const appName = "ConsumerApp"
 
 // We pull these out so we can set them with LDFLAGS in the Makefile
 var (
-	NodeDir      = ".meshd"
-	Bech32Prefix = "mesh"
+	NodeDir      = ".bcd"
+	Bech32Prefix = "bbnc"
```

Comment:
Nit: This could be `cons` or `consumer` for clarity.

Reply:
Hmm I prefer `bbnc` as `cons` seems reserved for consensus addresses in Cosmos SDK and `consumer` is a bit long. Wdyt?